resources/tutorial/.gitignore

```diff
@@ -1,2 +1,5 @@
 # Output folders
 out/*.csv
+.Rhistory
+.RData
+plots.R
```

Comment:
Missing new line at the end of file. Please consider install the pre-commit hooks for an automatic fix at commit stage.